src/components/Modal/LoginModal.js

```diff
@@ -65,7 +75,13 @@ const LoginModal = () => {
                 </span>
                 Continue with Facebook
               </button>
-              <button className="continuegoogle-btn">
+              <button
+                onClick={() =>
```

Comment:
its better to create function in constant and use it here (dont use nameless functions in this case)